File: wrappers/go/ipaship.go
package ipaship

import "fmt"

type Client struct {
	APIKey string
}

func NewClient(apiKey string) *Client {
	return &Client{APIKey: apiKey}
}

func (c *Client) Audit(filePath string) error {
	fmt.Printf("Auditing %s via ipaship.com...\n", filePath)
	return nil
}

File: wrappers/php/src/IPAShip.php
<?php
namespace IPAShip;

class IPAShipClient {
    private $apiKey;

    public function __construct(string $apiKey) {
        $this->apiKey = $apiKey;
    }

    public function audit(string $filePath) {
        echo "Auditing {$filePath} via ipaship.com...\n";
    }
}